HW 1/style.css

```diff
@@ -0,0 +1,54 @@
+#myBtn {

```

Comment:
it's better to use styles via classes and not via id's. It's not breaking but could lead to a potential problems.